docs/root/version_history/current.rst

```diff
@@ -10,6 +10,8 @@ Changes
   are applied to using :ref:`HTTP headers <config_http_filters_fault_injection_http_header>` to the HTTP fault filter.
 * http: fixed a bug where the upgrade header was not cleared on responses to non-upgrade requests.
   Can be reverted temporarily by setting runtime feature `envoy.reloadable_features.fix_upgrade_response` to false.
+* listener: added additional draining filter chain stat for :ref:`listener manager <config_listener_manager_stats>`.
+* listener: added intelligent update path which doesn't close connection if the corresponding network filter chain is equivalent during the listener update.
```

Comment:
```suggestion
* listener: added intelligent update path which doesn't close connections if the corresponding network filter chain is equivalent during the listener update.
```

Reply:
Please remove the docs/version history from this change though until we get to the final change in which this is actually turned on. In this change it's not turned on, right? If it is turned on in this change we need to have a runtime kill switch in this change.

Reply:
added